src/NativeScript/inspector/GlobalObjectInspectorController.cpp

```diff
@@ -125,7 +125,19 @@ GlobalObjectInspectorController::~GlobalObjectInspectorController() {
 }
 
 void GlobalObjectInspectorController::registerDomainDispatcher(WTF::String domainIdentifier, JSC::JSCell* constructorFunction) {
-    std::unique_ptr<DomainInspectorAgent> domainInspectorAgent = std::make_unique<DomainInspectorAgent>(m_globalObject, domainIdentifier, constructorFunction);
+    AgentContext baseContext = {
```

Comment:
I have a suspicion that the reason you can't do it inline is because the  `AgentContext&` parameter isn't marked as `const` in the constructor: https://gist.github.com/anonymous/306cfb0e33be4c2e9855.